provision/acc_provision/acc_provision.py

```diff
@@ -1029,24 +1050,26 @@ def config_validate_preexisting(config, prov_apic):
 
 def calico_config_validate_preexisting(config, prov_apic):
     try:
+        apic = None
         if prov_apic is not None:
             apic = get_apic(config)
             if apic is None:
                 return False
-        aep_name = config["aci_config"]["aep"]
-        physical_domain_name = config["aci_config"]["physical_domain"]["domain"]
+        aep_name = config["aci_config"]["cluster_l3out"]["aep"]
         vrf_tenant = config["aci_config"]["vrf"]["tenant"]
         vrf_name = config["aci_config"]["vrf"]["name"]
         vrf_dn = config["aci_config"]["vrf"]["dn"]
         l3out_name = config["aci_config"]["l3out"]["name"]
-        phys_dom = apic.get_phys_dom(physical_domain_name)
-        if phys_dom is None:
-            err("Physical Domain %s not created on the APIC. Please create the Physical Domain and try again" % physical_domain_name)
-            return False
         aep = apic.get_aep(aep_name)
         if aep is None:
             err("AEP %s not created on the APIC. Please create the AEP and try again" % aep_name)
             return False
+        check_local_asn = apic.get_local_asn()
+        aci_as_number = config["aci_config"]["cluster_l3out"]["bgp"]["peering"]["aci_as_number"]
+        if str(aci_as_number) != check_local_asn:
+            err("aci_as_number %s provided in the input file does not match the BGP route reflector asn %s on the APIC" %
```

Comment:
Just add the following to the comment. "This check is only made if -a option is used. So ensure the calico manifests that are generated from this step with -a are used instead of previous ones you may have had"